lib/engine/game/g_1840/step/acquisition_auction.rb

```diff
@@ -22,6 +22,10 @@ def actions(_entity)
             actions
           end
 
+          def log_with_player(entity)
```

Comment:
This method doesn't log anything, so its name is misleading. How about `name_with_player`?